.github/workflows/analysis-of-endpoint-connections.yml

```diff
@@ -1,12 +1,6 @@
 on:
   workflow_dispatch:
-  pull_request:
-    types:
-      - opened
-      - synchronize
-    paths:
-      - 'src/main/java/**'
-      - 'src/main/webapp/**'
+  push:
```

Comment:
Do we want to run this on push or should we have an open PR for this to run?

Also, is it normal for the H2 tests / Junit results / Java Architecture tests to run under your workflow now?

![image](https://github.com/user-attachments/assets/07875cfa-2159-4a4e-92b9-78e3c0ecee49)

![image](https://github.com/user-attachments/assets/6a5cea12-4a7b-4a54-8e8f-21a1c1ec511f)

Reply:
I think it makes sense to also run the analysis without an open PR, as the gathered information is useful on a single branch. IMO, it makes sense to be able to see on develop what endpoints are not used and what REST calls don't have a matching endpoint.

Regarding the grouping of the H2 tests / Junit results / Java Architecture tests, Github groups the actions by the event that triggers them, which makes them appear together.

Reply:
Also, there are always different jobs that appear there, and the display of jobs on GitHub seems to be buggy in general.